src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -46,18 +45,29 @@ public static boolean areSameHtml(String expected, String actual, boolean isPart
     
     private static boolean assertSameHtml(String expected, String actual, boolean isPart,
                                           boolean isDifferenceToBeShown) {
-        String processedExpected = convertToStandardHtml(expected, isPart);
+        String processedExpected = expected.replace("\r", "");
```

Comment:
@wkurniawan07 if you are sure that we are using this *only* for the CR+LF case, try to be more specific by using: ```replace("\r\n", "\n");```